modules/nodes-pods-autoscaling-policies.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-policies_{context}"]
+= Scaling policies
+
+The `autoscaling/v2beta2` API allows you to create _scaling policies_ that control how the {product-title} horizontal pod autoscaler (HPA) scales Pods. Scaling policies allow you to restrict the rate that HPAs scale Pods up or down by setting a specific number or specific percentage to scale in a specified period of time. You can also define a _stabilization window_ which uses previously computed desired states to control scaling if the metrics are fluctuating. You can create multiple policies for the same scaling direction and determine which policy is used first, based on the amount of change. You can also restrict the scaling by timed iterations. The HPA scales pods during an iteration, then performs scaling, as needed, in further iterations. 
+
+By default, the scaling policies are implemented when the number of Pods associated with an HPA is greater than 40.
```

Comment:
@joelsmith Is this true? The upstream _seemed_ to suggest this. Assuming it exists, is this a hard-coded value or can users set?

Reply:
What upstream docs meant was that since there were two policies (one for a number of pods, and one for a percentage of pods), the policy that results in the largest change will be used. The number-of-pod policy specified 4 pods, so it would be larger in any case where the total number of pods was less than 40. The percentage policy would be ignored in those cases. Once the number of pods gets above 40, then 10% of the number would be >= 4, so the percentage policy would be used.

Imagine a scenario with a pod-number policy of 1, and a percentage policy of 25%.  In that case, if there were 1, 2, or 3 pods, the pod-number policy of 1 would be bigger than 0.25, 0.5 and 0.75, so the policy of 1 would be used. At 4 pods, they would both be 1. At more than 4 pods, the 25% policy would be larger than 1, so it would be used from there on out. Of course, I'm ignoring the rounding.